webpack/assets/javascripts/foreman_tools.js

```diff
@@ -44,6 +44,11 @@ export function activateTooltips(el = 'body') {
                                    }
                               });
   el.find('*[title]').not('*[rel]').tooltip({ container: 'body' });
+
+  // adding this to remove displayed tooltips for react-initiated
+  // calls to this function (no page:restore fired).
+  $('.tooltip.in').remove();
```

Comment:
this is here because react components that change leave their opened tooltips open forever.
the solution: call initialize_tooltips whenever the component changes, and there remove the already opened tooltips.
also, i think it makes sense to remove existing tooltips when initializing.